<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="videoListWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="materialListWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openVideoBtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>导入视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSucaiBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>添加素材</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addToTrackBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>添加到轨道</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>290</y>
      <width>480</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playBtn">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>310</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/icons/play.ico</normaloff>:/icons/icons/play.ico</iconset>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="textLabel">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>219</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="scaleWidget" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>390</y>
      <width>480</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="videoTrackWidget" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>430</y>
      <width>480</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="tupianWidget" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>490</y>
      <width>480</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>370</y>
      <width>480</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QVideoWidget" name="videoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="importBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>导入素材</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteSucaiBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>删除素材</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zimuOPenBtn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>610</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>导入字幕</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zimuLineEdit">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>610</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>字幕地址</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zimuClearBtn">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>610</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>清除字幕</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outBtn">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>310</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>导出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addTextBtn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>添加文字</string>
    </property>
   </widget>
   <widget class="QWidget" name="subTextWidget" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>550</y>
      <width>480</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mergeBtn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>320</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>合并</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuTools">
     <property name="title">
      <string>Tools</string>
     </property>
     <addaction name="actionPick"/>
     <addaction name="actionClip"/>
    </widget>
    <addaction name="actionExport"/>
    <addaction name="menuTools"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionPick">
   <property name="text">
    <string>pick</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionClip">
   <property name="text">
    <string>clip</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
